lld/test/wasm/unresolved-symbols.s

```diff
@@ -1,30 +1,34 @@
-# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %s -o %t1.o
+# RUN: split-file %s %t
```

Comment:
TIL split-file is a thing...

Reply:
Yeah its kind of nice!   One downside is that the test files themselves are less independently useful, but thats seems like a minor things compared to the hassle of splitting out secondary files into the `Input/` diretory.